subsys/net/lib/nrf_cloud/src/nrf_cloud_info.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <net/nrf_cloud.h>
+#if defined(CONFIG_MODEM_INFO)
+#include <modem/modem_info.h>
+#endif
+#if defined(CONFIG_MODEM_JWT)
+#include <modem/modem_jwt.h>
+#endif
+#include "nrf_cloud_transport.h"
+#include "nrf_cloud_client_id.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(nrf_cloud_info, CONFIG_NRF_CLOUD_LOG_LEVEL);
+
+#if defined(CONFIG_MODEM_INFO) && defined(CONFIG_NRF_CLOUD_VERBOSE_DETAILS)
+static struct modem_param_info mdm_param;
+
+static int get_modem_info(void)
```

Comment:
nit: maybe `print_modem_info` instead of `get_modem_info`, not that it matters a ton

Reply:
I've refactored it. This is gone now.